### ISSUE TITLE
Rebuilding Shelf & Drawer

### ISSUE THREAD

**Issue by narekboghozian** [Triton-AI/evGoKart-mechanical-system#1]
We're making this out of Delron as per regulations. Do we have a new mounting pattern?

**Comment by Arjun-Naageshwaran** [comment]
Hey my name is Arjun, I am part of Mechanical team and I am working on the drawer. I consulted Matthew, and he agreed that Delrin isn't strong enough. We are using a 1/16" aluminum plate which we are going to bend using the bending press. I made a proposal for that purchase, but I can't seem to contact Matthew over Basecamp.

**Comment by narekboghozian** [comment]
Hi Arjun, I emailed Matthew so he should hopefully see this soon.

> Delrin isn't strong enough

I've been reading the opposite, that Delrin is stronger than Acrylic? @jacksilb did you also discuss this with Matthew?

> I made a proposal for that purchase...

Can you link that proposal here? 
What kind of work is going to go in to that construction?
Do you know if we have the parts already?

**Comment by Arjun-Naageshwaran** [comment]
I discussed it with Matthew, and he was saying that Delrin is not a preferable alternative to aluminum. From what I understand, I don't think he even considered acrylic. 

My proposal isn't for the whole setup, just the piece of aluminum needed for the drawer. Matthew told me to find the right piece of aluminum from the McMaster-Carr website. I based dimensions on the drawer CAD, and talked to Chris Cassidy and Steve from the machine shop about their bending presses. We need 1/16" aluminum sheets which we will then laser cut and bend.

I did not make these decisions, which is why I wanted to consult Matthew, but again I could not contact him on Basecamp.

**Comment by jacksilb** [comment]
We need to top self too no?
We can't have Acrylic in the evGoKart.

Can we have a quick design review to do over the CAD/ design?
Otherwise we are going with delrin to not slow down the project. I have been talking about this for weeks. Friday was the dead-line to have the aluminum shelves 
Please share the design with us for comments.
We need the shelves done one way or the other by next week. Once we start to put them in the robot will be down.

**Comment by jacksilb** [comment]
https://www.refugeforums.com/threads/delrin-vs-acrylic.870866/

I thought I posted something like this already.

I believe we discussed aluminum as the ideal solution, but if we can't get it done this week. We will need to go with delrin.

The MAE 156B team can help.

thanks
-Jack

**Comment by jacksilb** [comment]
https://sawmillcreek.org/showthread.php?139045-Plastic-Comparisons-of-Delrin-Acrylic-and-Lexan

**Comment by jacksilb** [comment]
https://hackaday.com/2016/03/03/materials-to-know-acetal-and-delrin/

Delrin, Acetal, and its many trade names is a material properly known as Polyoxymethylene or POM. It is one of the strongest plastics and is a good go-to material when you want the best properties of plastic, and don’t need the full strength of a metal part. It was originally formulated to compete with Zinc and Aluminum castings after all.

**Comment by narekboghozian** [comment]
> I discussed it with Matthew, and he was saying that Delrin is not a preferable alternative to aluminum. From what I understand, I don't think he even considered acrylic.

@jacksilb he's not saying acrylic is stronger than Delrin. He's saying that Matthew thought neither was strong enough. I don't really believe acrylic is strong enough either, given how much stress & vibration there will be.

If we feel that Delrin still not strong enough, let's just reinforce it. But I don't see why we couldn't use aluminum extrusions to make that drawer structure in the first place though.

Regarding the form factor, the laser cutters had trouble carrying out large cuts without failing, so let's break up the shelf into 2, 3, or 4 segments that will be easier to cut. That's a quick change that will take less time than re-doing the entire thing.

**Comment by Arjun-Naageshwaran** [comment]
Hey sorry for the late response, I was traveling this weekend. @jacksilb I sent you the CAD screenshots and the selection of aluminum sheet I made from the McMaster-Carr website. Is that suitable? Or should we start with the Delrin?

**Comment by jacksilb** [comment]
Please review with the MAE students the design ASAP.
The window is closing to use Aluminum


Thanks,
-Jack
----

**Comment by narekboghozian** [comment]
@Arjun-Naageshwaran  can you post that here?

**Comment by GriffinBorn** [comment]
Hello guys my name is Jack Griffin, I am from the MAE 156B team,
I can see you guys are still somewhat deciding between Aluminum and Delrin, I would like to add that I did do some analysis and found that Delrin should hold up well, and if necessary we could add some aluminum supports to help aleviate stress due to vibrations. Also it was actually my MAE professor, Professor Delson, who actually recommended the use of Delrin during our early design proposals last quarter when discussing an alternative for Acrylic that is stronger and not as expensive as Aluminum. Much of the simulations was also based on impact since we were looking at shatter resistance and found Delrin to hold well. Here is a table that compares the specs of the materials. 

![image](https://user-images.githubusercontent.com/54219548/163755758-c4cfd654-8f8f-4be9-a2ad-aac4fb339ba5.png)

If you guys have any more questions, or need assistance with machining please feel free to message me and we can figure things out.

**Comment by Arjun-Naageshwaran** [comment]
@jacksilb @narekboghozian 
Hello, after reviewing the attached CAD models for dimensions and bending style, I have selected the attached aluminum sheet. We have to use the 24" x 48" aluminum sheet that is 0.062" thick, since that thickness is the thickest that will fit into the bending press.

![Screenshot (120)](https://user-images.githubusercontent.com/103783265/163764753-20c0cd45-77d8-4b66-ba21-344c00ffdf8b.png)
![Screenshot (121)](https://user-images.githubusercontent.com/103783265/163764757-6e2bb212-4e6e-4549-928c-8f54c7817b39.png)
![Screenshot (122)](https://user-images.githubusercontent.com/103783265/163764760-eb53fc77-3e9c-4d47-93e6-ad75df75fad8.png)

**Comment by Arjun-Naageshwaran** [comment]
The dimensions of the drawer cutout are 34.68" x 14.118". As I said above, the 24" x 48" aluminum sheet will cover these dimensions. It is $111.64 which is more expensive than the Delrin

**Comment by Arjun-Naageshwaran** [comment]
Apologies for the delay on this, I sent it to @jacksilb earlier, but wanted to run it by Matthew too. Although I have not been able to contact him.

**Comment by Arjun-Naageshwaran** [comment]
It is also worth noting that Matthew was staunchly against Delrin for this purpose. I have not looked into either extensively enough.

**Comment by GriffinBorn** [comment]
I understand this is for the inside drawer, will the top shelf also be cut from aluminum? Remember all forms of acrylic on the kart needs to be replaced.
Also will 0.062" be strong enough to hold up the electronics? Because at least to me that seems very thin and will very easily bend under the weight of the electronics. Have you done force simulations on the drawer? Because I was under the assumption that if we would use aluminum we would at least be 1/8" which is twice this thickness.

**Comment by GriffinBorn** [comment]
Ok so my filing cabinet drawer is about 0.062" thick aluminum so this thickness of aluminum could work, however, my drawer is much smaller than what the drawer on the kart would be and is held together from all sides, so some static stress simulations would be nice to see before creating this piece to ensure it will support all the weight. 
Also, I updated the Fusion part so that it is now of the correct aluminum material.

**Comment by GriffinBorn** [comment]
Also when you do a simulation, make sure to update the body thickness. I had a look at the Fusion cad and right now it is about 3x the thickness of the actual aluminum you are planning on using. 

Regarding the simulations, I recommend making 0.0001" cuts in the shape of the mounts or electronics that will be placed on the shelf itself, then applying the force of their weight on those cut sections.

Here is an example how I did it on a plate I created last year (this was made in Solidworks but it should be the same in Fusion):

This is what it looks like normally:
![image](https://user-images.githubusercontent.com/54219548/163771993-343ace3f-7822-403e-8b38-e4d6066721df.png)

This is what it looks like with the small cuts:
![image](https://user-images.githubusercontent.com/54219548/163771776-021fa6a1-8b55-44b7-9401-7ef2aa3bdcbc.png)

This is what it looks like when I apply force to those certain sections:
![image](https://user-images.githubusercontent.com/54219548/163772144-86d103c2-b1c4-4147-8347-2d07c3045883.png)

You should be able to do something similar and much easier with a couple of squares.

**Comment by GriffinBorn** [comment]
Doing a model frequency analysis may be useful as well.

**Comment by Arjun-Naageshwaran** [comment]
Those are all solid suggestions. The reasoning behind 1/16" is that it is simply the maximum thickness both bending presses we have access to can bend. At this point, since Matthew has been out of communication, Delrin might have to be our option. I've been trying to get his advice on this since last Tuesday to fill the gaps in my knowledge, but he is off Basecamp and not responding on discord.

**Comment by Arjun-Naageshwaran** [comment]
@jacksilb because Matthew hasn't responses, I guess we are going with Delrin? I can begin adjusting the CAD with supports accordingly.

**Comment by jacksilb** [comment]
Fellows, we need a design review soon and move on.
Let's do Delrin for now. Easy to laser cut. Please talk to @acastane and the MAE 156 team today and let me know what to buy. I can go to the plastic store tomorrow and assuming they have in stock get Delrin. @narekboghozian and @genesis already have a simple solution to add extra support. One simple bar across the bottom shelve should do it. We have the stock at home. We can do the cross bar today if someone can just cut and tap it.

In summary, what size of Delrin plates you want me to get and the thickness. Lets rock!

**Comment by Arjun-Naageshwaran** [comment]
I don't have any of the contact info for the MAE 156 team, does anyone on this thread have it?

**Comment by GriffinBorn** [comment]
I am one of the members of the 156 team, Adrian (one of the other MAE 156 team members) and I will be in Jack's lab tomorrow if you are able to come in and we can discuss things. Otherwise, I will provide our emails here and you can group email us:
- jsgriffi@ucsd.edu
- acastane@ucsd.edu
- d1qiao@ucsd.ed
- hkn013@ucsd.edu

**Comment by jacksilb** [comment]
Arjun, Jack, and MAE 156B team.
Let's move on and get Delrin.
We ran out of time to use aluminum.
Jack, as we discussed yesterday, please let me know the thickness and size of the Delrin/platic that fits at the MAE Design studio laser cutter.  I am planning to buy it in the next day or so. I will try to buy by phone then run there to get it. It is a 45 min drive easy way ...
Thank you,
-Jack

**Comment by Arjun-Naageshwaran** [comment]
@GriffinBorn I can't make it to the lab, but I will def email you.

**Comment by GriffinBorn** [comment]
@Arjun-Naageshwaran Do you have the sketches or DXF's for the shelf parts that we need to laser cut? We have the Delrin sheets in the lab so as soon as we have the sketches for the parts we can get them cut.

**Comment by jacksilb** [comment]
I believe Vlad and Colling said they have them too.

**Comment by colin-szeto** [comment]
[top plate and all parts bottom plate.zip](https://github.com/Triton-AI/evGoKart-mechanical-system/files/8558100/top.plate.and.all.parts.bottom.plate.zip) (out of date , see [below](https://github.com/Triton-AI/evGoKart-mechanical-system/issues/1#issuecomment-1109472719) for the correct zip files)

See attached file. Let me know if you have any questions

**Comment by jacksilb** [comment]
Nice.

Lets laser cut them as quickly as possible. Good timing to change them
before we start the test outside. We are behind the schedule. We have not
stop once we the robot ready for software development.


Thanks,
-Jack
----

**Comment by GriffinBorn** [comment]
So I tried to open these files and was wondering if they were made to fit the delrin sheets we have? Also the Top Plate dxf seems to not have any lines or holes; and then rev4Allparts file seems to have some overlap in some area. Is it supposed to be like this? 
Here are images of both; the top plate I filled to see if I can see the holes but there seems to not be any strokes/lines they would be blue if they were there. The other parts seem to overlap and have some weird lines.
![image](https://user-images.githubusercontent.com/54219548/165211331-59fa0ff2-48a7-4f3a-867a-5203e7a04962.png)
![image](https://user-images.githubusercontent.com/54219548/165211400-14318c56-b892-44da-b4ed-9816def76e11.png)
The dimensions also seem to be off, that could be a my side thing but I am not sure

**Comment by colin-szeto** [comment]
Hmm opening them in adobe illustrator may not be the best idea as the dxf were generated in AutoCAD. 

I don't know the dimensions of the delrin in the lab, could you provide them> 

![image](https://user-images.githubusercontent.com/29698074/165221442-fa52ccbf-ee0b-49e9-885d-94ec9fbdd4bc.png)

![image](https://user-images.githubusercontent.com/29698074/165221506-d89880a8-cf61-465b-b6b5-37320c745c0f.png)

^these are what they look like in autocad

**Comment by GriffinBorn** [comment]
Ah perfect, just wanted to make sure, I don't have AutoCAD on my computer so it is good that it looks good there. Here are the max dimensions that are allowed in the laser cutter, and I believe this matches the dimensions of the sheets we have. 
2' x 4'

**Comment by colin-szeto** [comment]
@GriffinBorn 

Here are the updated files. These have 20mm by 20mm hole patterns
[to github.zip](https://github.com/Triton-AI/evGoKart-mechanical-system/files/8561033/to.github.zip)
![image](https://user-images.githubusercontent.com/29698074/165251061-01bb5144-5ba8-4dfa-9993-f61e3853f1ca.png)
![image](https://user-images.githubusercontent.com/29698074/165251094-3da98e1b-8ea5-4303-8098-d8c4ea03f00b.png)

**Comment by GriffinBorn** [comment]
Would it be possible to do 15mm x 15 mm distance hole pattern in this area? 
![image](https://user-images.githubusercontent.com/54219548/165258482-5a49ef1b-9327-4804-905d-4918df37092d.png)

One of my other team members is mounting the vesc and says the center distance between holes for the VESC mount is 75 mm.

**Comment by colin-szeto** [comment]
![image](https://user-images.githubusercontent.com/29698074/165263305-2830ecbe-d93d-4778-828f-db95c6c204f1.png)
[top top plate with 15 by 15.zip](https://github.com/Triton-AI/evGoKart-mechanical-system/files/8561513/top.top.plate.with.15.by.15.zip)

The 15 by 15 hole pattern is attached above

**Comment by GriffinBorn** [comment]
Thanks, I'll see about getting these cut tomorrow then.

**Comment by GriffinBorn** [comment]
@colin-szeto I showed my teammate and he just replied to me saying he actually needs the outer holes for that section to be 15mm apart also if possible . If not he can figure out another way to mount, but it would make things easier if they were.

**Comment by colin-szeto** [comment]
@GriffinBorn 

![image](https://user-images.githubusercontent.com/29698074/165326697-53fb8e1d-9705-4423-be0f-addda60351b4.png)

[top top plate with 15 by 15 leddge outer.zip](https://github.com/Triton-AI/evGoKart-mechanical-system/files/8564222/top.top.plate.with.15.by.15.leddge.outer.zip)

**Comment by GriffinBorn** [comment]
Perfect thanks 👍

**Comment by Arjun-Naageshwaran** [comment]
@GriffinBorn I am so sorry, I have been super busy with midterms last week, and continue to be this week. Let me know if you need anything further with the drawer.

**Comment by GriffinBorn** [comment]
@narekboghozian Could you update and provide me the dxf files for this electrical stack so we can make it into Delrin. Also as a note from what I noticed when doing the main shelf, please update the dimensions for 0.25" material which is also around 6mm; I just noticed that the previous holes for slotting the parts together were very tight and I had to sand the holes down and it was because they weren't adjusted from the original 5mm thickness to the new 0.25" thickness. So if I could get those dxf files tonight or tomorrow morning so we can get those cut ASAP that would be great, thank you.
![image](https://user-images.githubusercontent.com/54219548/166180264-dd4fbc32-4a26-45e9-b727-00fa9cf51e26.png)
Image for reference of what I am talking about.
